opm/test_util/summaryComparator.hpp

```diff
@@ -26,6 +26,17 @@
 #include <algorithm>
 #include <string>
 
+
+// helper macro to handle error throws or not
+#define THROW_OR_NOT(type, message) \
```

Comment:
which is precisely what this macro does, no ? (through the if()). anyways, i'll rename..